roles/kubernetes/master/tasks/post-upgrade.yml

```diff
@@ -1,6 +1,29 @@
 ---
+- name: "Post-upgrade | stop kubelet on all masters"
+  service:
+    name: kubelet
+    state: stopped
+  delegate_to: "{{item}}"
+  with_items: "{{groups['kube-master']}}"
+  when: needs_etcd_migration|bool
+
+- name: "Post-upgrade | Pause for kubelet stop"
+  pause:
```

Comment:
Would this be better handled by a `wait_for` instead of pause? 
Maybe 
```
wait_for:
  host: localhost
  port: {{ kube_apiserver_insecure_port }}
  timeout: 180
```